Code/GraphMol/MolDraw2D/catch_tests.cpp

```diff
@@ -9406,16 +9407,24 @@ TEST_CASE("Github7036 - triple bond to wedge not right") {
   // ends in the wrong place when the incident bond is
   // a wedge.  Wedge to single bond included for visual
   // check that that isn't broken in the fix.
+  // Make sure it works for both coords generators.
+  // Tracking down issue 7620 demonstrated this was not
+  // always the case.
```

Comment:
This is ok as is, but it would have been a bit nicer to include a CXSMILES with coordinates (or a mol block) for a molecule with the layout that triggered the bug.

Reply:
This and #7739 are pretty much the same bug. I’ll send a fix in for that that merges this one in as well.  They will conflict if merged separately in any case. I’ll add the extra test case at that point as well. An algorithmic test will be fiddly- this is obviously wrong visually but less clear to pick out on the SVG.

Reply:
Scrap that - I've decided to leave them as 2 PRs as they are a bit different.  This one should probably be merged first, since it's already reviewed.  I expect there'll be a conflict when merging the 2nd.  Sorry for the mucking about.